.dockerignore

```diff
@@ -0,0 +1,14 @@
+.git/
+node_modules/
+dist/
+.github/
+configs/
+branding/
+scripts/
```

Comment:
Does this register with scripts ignored? I haven't tried this yet.

Reply:
Commands aren't automatically deployed to Discord anyway, so ignoring scripts doesn't affect anything.

I could add a command that runs on startup though.

Reply:
I'm thinking about this- perhaps leave it out. If we change commands, we should manually update IMO.